services/httpd/handler.go

```diff
@@ -464,15 +465,15 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *meta.
 	_, err := buf.ReadFrom(body)
 	if err != nil {
 		if h.Config.WriteTracing {
-			h.Logger.Print("write handler unable to read bytes from request body")
+			h.Logger.Print("Write handler unable to read bytes from request body.")
```

Comment:
Why the change in capitalization and the addition of periods? We don't generally include those in log messages to my knowledge.